libnavigation-ui/src/main/java/com/mapbox/navigation/ui/camera/DynamicCamera.java

```diff
@@ -42,20 +40,6 @@ public DynamicCamera(@NonNull MapboxMap mapboxMap) {
     this.mapboxMap = mapboxMap;
   }
 
-  @Override
-  public double tilt(@NonNull RouteInformation routeInformation) {
```

Comment:
Are we ok breaking SemVer here?

Reply:
BTW Metalava is ❌

Reply:
This isn't breaking semver, there's still the parent `SimpleCamera#tilt` available.